src/main/resources/templates/email/participantProjectSpecies/added/body.txt.ftl

```diff
@@ -0,0 +1,10 @@
+<#-- @ftlvariable name="" type="com.terraformation.backend.email.model.ParticipantProjectSpeciesAdded" -->
+${strings("notification.participantProjectSpecies.added.email.subject", participantName)}
+
+${strings("notification.participantProjectSpecies.added.email.body.1", speciesName, projectName)}
```

Comment:
Should this be `body.1` and `body.2` instead of `subject` and `body`? Same with the plaintext file for the edit event.

Reply:
Good catch!